state/index.js

```diff
@@ -0,0 +1,7 @@
+/**
+ stores the users in each grid
+ * @type {Object.<string, Set>}
+ */
+export const grids = {};
```

Comment:
We should consider removing the geohash from the grids set once it has 0 users.

Reply:
Since you intend to move to redis for the actual storage eventually, you can skip extra work on the local temporary store for now as it'll become unnecessary then.